config/application.rb

```diff
@@ -19,6 +19,13 @@ class Application < Rails::Application
 
     routes.default_url_options[:host] = Figaro.env.domain_name
 
+    config.action_mailer.default_options = {
+      from: Mail::Address.new.tap do |mail|
+        mail.address = Figaro.env.email_from
+        mail.display_mail = Figaro.env.email_from_display_name
```

Comment:
Big whoops, my suggestion had a typo in the method name, which got copied here:

```
NoMethodError: undefined method `display_mail=' for #<Mail::Address:70174593893800 Address: |no-reply@login.gov| >
```

```suggestion
        mail.display_name = Figaro.env.email_from_display_name
```